pkg/controllers/disruption/controller.go

```diff
@@ -230,14 +230,14 @@ func (c *Controller) launchReplacementNodeClaims(ctx context.Context, m Method,
 	stateNodes := lo.Map(cmd.candidates, func(c *Candidate, _ int) *state.StateNode { return c.StateNode })
 
 	// taint the candidate nodes before we launch the replacements to prevent new pods from scheduling to the candidate nodes
-	if err := c.requireNoScheduleTaints(ctx, true, stateNodes...); err != nil {
+	if err := c.requireNodeClaimNoScheduleTaint(ctx, true, stateNodes...); err != nil {
```

Comment:
```suggestion
	if err := c.requireNoScheduleTaint(ctx, true, stateNodes...); err != nil {
```

Do we need to be specific about NodeClaim/Machine now that we just have one concept?